java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -824,6 +824,27 @@ public void sadd_srem_scard_smembers_key_with_non_set_value(BaseClient client) {
         assertTrue(e.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void smismember(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+
+        assertEquals(2, client.sadd(key1, new String[] {"1", "2"}).get());
+        assertArrayEquals(
+                new Boolean[] {true, false}, client.smismembmer(key1, new String[] {"1", "3"}).get());
+        // empty set
```

Comment:
Not empty set the key does not exist in this example

Reply:
empty set = key does not exist
You can't create an empty set; if you remove all elements from it - key would be deleted